src/selectedtests/test_mappings/mappings.py

```diff
@@ -197,16 +204,16 @@ def create_mappings(
         :param repo: The repo that contains the source code for the evergreen project.
         :param source_re: Regex pattern to match changed source files against.
         :param test_re: Regex pattern to match changed test files against.
-        :param start_date: The date at which to start analyzing commits of the project.
-        :param end_date: The date up to which we should analyze commits of the project.
+        :param after_date: The date at which to start analyzing commits of the project.
+        :param before_date: The date up to which we should analyze commits of the project.
         :param project: The name of the evergreen project to analyze.
         :param branch: The branch of the git repo used for the evergreen project.
         :return: An instance of the test mappings class
         """
         file_intersection = defaultdict(lambda: defaultdict(int))
         file_count = defaultdict(int)
 
-        LOGGER.debug(f"searching from {start_date} unil {end_date}")
+        LOGGER.debug(f"searching from {after_date} unil {before_date}")
```

Comment:
Good point, I had forgotten this log line. It's actually logging the same info as [this other log line](https://github.com/mongodb/selected-tests/pull/17/files#diff-53482f04effe60f8729a630eef4f0a7aR48-R49), so I'm removing it. (I hadn't noticed it since I wasn't in verbose mode.)